visidata/sheets.py

```diff
@@ -303,8 +303,9 @@ def loader(self):
         self.rows = []
         try:
             with vd.Progress(gerund='loading', total=0):
+                max_rows = self.options.max_rows
                 for i, r in enumerate(self.iterload()):
-                    if self.precious and i > self.options.max_rows:
+                    if self.precious and i == max_rows:
```

Comment:
Let's stick with `>`.  This `==` actually stops a row earlier.

Reply:
With `>` I'm getting an extra row:
```
seq 5 | vd -f txt --max-rows=3 -b -o -   # with >
saving 1 sheets to - as txt
1
2
3
4
```
But with `==` it seems right to me:
```
seq 5 | vd -f txt --max-rows=3 -b -o -  # with ==
saving 1 sheets to - as txt
1
2
3
```

```
seq 10 | vd --header=0 -f tsv --max-rows=3 -b -o - # with ==
saving 1 sheets to - as tsv

1
2
3
```